### ISSUE TITLE
download track as gpx and kml

### ISSUE THREAD

**Issue by Courgetteandratatouille** [c2corg/v6_api#665]
For apps we need to download tracks as gpx and kml :)

**Comment by brunobesson** [comment]
Sorry, I don't understand this issue. Tracks and outings can already be exported in GPX and KML formats.

**Comment by Courgetteandratatouille** [comment]
Yes but it's not downloadable directly from apps, we need to go to the web site and click on the button. 

We can do this on the app, but it's not userfriendly, it's better to launch directly the download from the app. And app need to be separe from the ui the more as possible from the ui.

**Comment by brunobesson** [comment]
My bad, I didn't see this was an API issue and not an UI one.

**Comment by asaunier** [comment]
Python tools that might be used to build KML or GPX files:

KML
* http://simplekml.readthedocs.io/en/latest/
* https://pypi.python.org/pypi/fastkml (see http://fastkml.readthedocs.io/en/latest/usage_guide.html#build-a-kml-from-scratch)
* perhaps also https://pythonhosted.org/pykml/tutorial.html

GPX
*

It seems also possible to export a PostGIS geometry to KML (but some parts are missing to make it a valid KML file): https://postgis.net/docs/ST_AsKML.html
